src/containers/PipelineRun/PipelineRun.js

```diff
@@ -225,13 +272,52 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
 
     return (
       <>
-        <RunHeader
-          lastTransitionTime={lastTransitionTime}
-          loading={loading}
-          runName={pipelineRunName}
-          reason={pipelineRunReason}
-          status={pipelineRunStatus}
-        />
+        {/* prettier-ignore */ (showRebuildNotification && !showRebuildNotificationLogsURL) && (
+          <ToastNotification
+            data-testid="rebuildfailurenotification"
+            lowContrast
+            subtitle=""
+            title={showRebuildNotificationMessage}
+            kind={showRebuildNotificationKind}            
+          />
+        )}
+
+        {/* prettier-ignore */ (showRebuildNotification && showRebuildNotificationLogsURL) && (
+          <ToastNotification
+            data-testid="rebuildsuccessnotification"
+            lowContrast
+            subtitle=""
+            title={showRebuildNotificationMessage}
+            kind={showRebuildNotificationKind}
+            caption={
+              <Link id="newpipelinerunlink" 
+                to={showRebuildNotificationLogsURL}
+                onClick={ () => this.setShowRebuildNotification(false)}
```

Comment:
if you're navigating away and this component is unmounted the state will be re-initialised next time you return anyway

Reply:
Hmm without this added, on the rebuilt PipelineRun this caption is presented

Reply:
my bad, since you're navigating from PipelineRun to another PipelineRun the component isn't unmounted just updated